.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,21 @@
-<!-- For ugly or incorrect code issues: Please use this template.
+<!--
+
+BEFORE SUBMITTING AN ISSUE:
+
+1. Search for your issue on GitHub: https://github.com/prettier/prettier/issues
+   A large number of opened issues are duplicates of existing issues.
+   If someone has already opened an issue for what you are experiencing,
+   you do not need to open a new issue.
```

Comment:
How about adding an recommendation to 👍 the issue so we see people want it?

Reply:
Good idea